utils/config/i18n.ts

```diff
@@ -2,10 +2,123 @@ import { LocaleMessages } from 'vue-i18n'
 import MarkdownIt from 'markdown-it'
 import commonData from '@/locales/all_lang.json'
 
+export const langsFlags = [
+  {
+    value: 'en',
+    flag: '🇬🇧',
+    label: 'English',
+  },
+  // {
+  //   value: 'bn',
+  //   flag: '🇧🇩',
+  //   label: 'বাংলা',
+  // },
+  {
+    value: 'de',
+    flag: '🇩🇪',
+    label: 'Deutsch',
+  },
+  // {
+  //   value: 'cn',
+  //   flag: '🇨🇳',
+  //   label: '中文',
+  // },
+  // {
+  //   value: 'cz',
+  //   flag: '🇨🇿',
+  //   label: 'Česky',
+  // },
+  {
+    value: 'es',
+    flag: '🇪🇸',
+    label: 'Español',
+  },
+  {
+    value: 'fr',
+    flag: '🇫🇷',
+    label: 'Français',
+  },
+  // {
+  //   value: 'id',
+  //   flag: '🇮🇩',
+  //   label: 'Indonesia',
+  // },
+  // {
+  //   value: 'jp',
+  //   flag: '🇯🇵',
+  //   label: '日本語',
+  // },
+  // {
+  //   value: 'ko',
+  //   flag: '🇰🇷',
+  //   label: '한국어',
+  // },
+  // {
+  //   value: 'nl',
+  //   flag: '🇳🇱',
+  //   label: 'Vlaams',
+  // },
+  // {
+  //   value: 'pl',
+  //   flag: '🇵🇱',
+  //   label: 'Polski',
+  // },
+  // {
+  //   value: 'pt',
+  //   flag: '🇵🇹',
+  //   label: 'Português',
+  // },
+  // {
+  //   value: 'sk',
+  //   flag: '🇸🇰',
+  //   label: 'Slovenčina',
+  // },
+  // {
+  //   value: 'tu',
+  //   flag: '🇹🇷',
+  //   label: 'Türkçe',
+  // },
+  // {
+  //   value: 'ur',
+  //   flag: '🇵🇰',
+  //   label: 'اردو',
+  // },
+  // {
+  //   value: 'vt',
+  //   flag: '🇻🇳',
+  //   label: 'Tiếng Việt',
+  // },
+  // {
+  //   value: 'ru',
+  //   flag: '🇷🇺',
+  //   label: 'Русский',
+  // },
+  // {
+  //   value: 'ua',
+  //   flag: '🇺🇦',
+  //   label: 'Українська',
+  // },
+  // {
+  //   value: 'it',
+  //   flag: '🇮🇹',
+  //   label: 'Italiano'
+  // },
+  // {
+  //   value: 'hi',
+  //   flag: '🇮🇳',
+  //   label: 'हिन्दी'
+  // }
+]
+
 const md = MarkdownIt({
   breaks: false,
 })
 
+export const getLangsFlags = () => langsFlags
+
+export const getUserFlag = (userLang: string) =>
+  langsFlags.find((lang) => lang.value === userLang)?.flag || langsFlags[0].flag
```

Comment:
Why also flag?

Reply:
updated

Reply:
?

Reply:
I think `getUserFlag ` could stay in its original place.